modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/util/constants.js

```diff
@@ -12,14 +12,41 @@
  * details.
  */
 
+/**
+ *	Cart implementation constants
+ */
 export const DEFAULT_ORDER_DETAILS_PORTLET_ID =
 	'com_liferay_commerce_order_content_web_internal_portlet_' +
 	'CommerceOpenOrderContentPortlet';
-
 export const DISCOUNT_LEVEL_PREFIX = 'discountPercentageLevel';
-
 export const ORDER_DETAILS_ENDPOINT = '/pending-orders';
-
 export const ORDER_UUID_PARAMETER = 'commerceOrderUuid';
-
 export const WORKFLOW_STATUS_APPROVED = 0;
+
+/**
+ *	Cart component types keys constants
+ */
+export const CART = 'Cart';
+export const HEADER = 'Header';
+export const ITEM = 'Item';
+export const ITEMS_LIST = 'ItemsList';
+export const ITEMS_LIST_ACTIONS = 'ItemsListActions';
+export const OPENER = 'Opener';
+export const ORDER_BUTTON = 'OrderButton';
+export const SUMMARY = 'Summary';
+
+/**
+ * 	Cart labels keys constants
+ *
+ * 	These strings are not used as language keys,
+ * 	but rather to both document and override language keys.
+ *
+ * 	@see ./labels.js
+ */
+export const ADD_PRODUCT = 'Add a product to the cart';
```

Comment:
Just a thought, but you wouldn't need this comment if the values didn't _look like_ language keys. ie. if they had values like:

```js
export const ADD_PRODUCT = 'ADD_PRODUCT';
```

Reply:
Agreed, in principle... it's just that these lang keys constants were used also as part of the interface of the component for a customer to be able to override them. For instance:
```
const labels = {
    'Add a product to the cart': 'Aggiungi un prodotto al carrello',
    ...
};

<MiniCart labels={labels} ... />
```

And the quick approach that came to my mind was to use the same one I had seen somewhere else from past experiences, just to make it intuitive. Else, I would have needed to constant names in the code also as a "dictionary/map" for labels, which in turn might have needed another explanation about what they stood for, and ... you see where I'm going. Does it make sense? :o

Reply:
Yep, all good.